### PR TITLE
Add Christmas alternative to benzin roaches

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

It's Vietnam in maintenance tunnels today.

Added those bad boys to the game, there is a bunch of them pre-spawned in maintenance (three times more likely to spawn than a mouse), and infestation event always spawn them.

image

Also it's a little noisy first few minutes since game starts. Turf wars, see.

In the wild:
image

## Why It's Good For The Game

It's hilarious.

## Testing

Ran a local server, spawned infestation events.

## Changelog
:cl:
add: Added christmas-themed alternative to benzin roaches.
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
image
tfw can't merge this because its broke

File: code/modules/mob/living/carbon/superior_animal/elf.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/mob/living/carbon/superior_animal/elf
+	name = "arsonistic elf"
```
Comment (TheShown911) [inline code/modules/mob/living/carbon/superior_animal/elf.dm:2]:
```suggestion
	name = "fueltide elf"
```

**Comment by TheShown911** [comment]
Assuming direct control

**Comment by drexample** [comment]
NOOOO THIS IS LOW ARPEEE REVERT IMMEDIATELY

**Comment by drexample** [comment]
Ahem. Merry Christmas, everyone.

**Pull request merged**